docs/Way-of-Working/UbiquitousLanguage.md

```diff
@@ -0,0 +1,227 @@
+# Ubiquitous Language
+
+For clarity and consistency, this document defines key terms used within our team across various domains (e.g., Data
+Science, IT).
+
+The goal is to provide a list of commonly used terms and terms that are good to know. It is not to a full reference of
+all existing terms.
+
+Terms can and have been used 'as is' from sources mentioned at the end of this document.
```

Comment:
I'm not sure it is helpful to duplicate data from elsewhere (can't we just link then?). This list seems a little long. Can't we limit it to language we actually encouter/use?

Reply:
I second the remark of Anne. I think it would be a good idea to only add terms to this list that we encounter, otherwise the items on the list seem a bit random.